docs/user/dashboard/lens.asciidoc

```diff
@@ -319,7 +319,12 @@ You can also change the index pattern for a single *Layer*.
 [[why-my-field-x-is-missing-from-the-fields-list]]
 ===== Why is my field X missing from the fields list?
 
-*Lens* does not support the visualization of full-text fields, therefore it is not showing them in the data summary.
+If your field is not in the `Available fields` list in *Lens* there may be multiple reasons, here described:
+* the field is a full-text field, therefore it is not showing in the data summary.
+* the field is a Geo field, therefore it is not showing in the data summary prior to v.7.14.0
+* the mapping is not supported by index-patterns, for instance for fields in `flattened` objects
```

Comment:
Data analysts are unfamiliar with the concept of mapping. This statement is more for admins. 

I would reword this statement so that it applies to analysts, or remove it.